lib/completions/endpoints/anthropic.rb

```diff
@@ -38,6 +38,19 @@ def default_options(dialect)
 
           options = { model: mapped_model, max_tokens: max_tokens }
 
+          if llm_model.lookup_custom_param("enable_reasoning")
+            reasoning_tokens = llm_model.lookup_custom_param("reasoning_tokens").to_i
```

Comment:
```suggestion
            reasoning_tokens = llm_model.lookup_custom_param("reasoning_tokens").to_i.clamp(100, 65_536)
```

Using clamp here should keep the values in the range you want in a more succinct way